### PR TITLE
Updated package name

### PR DESCRIPTION
**Pull request by akashdabhi03**

### CODE REVIEW

**Review by ankitk20 — APPROVED** [review]

File: app/src/main/config/settings_pedro.plist [hunk 1]
```diff
@@ -118,7 +118,7 @@
 	<string>false</string>
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```
Comment (mathewnik90) [inline app/src/main/config/settings_pedro.plist:121]: What is this change?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,36 +1,30 @@
...
 
-    <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.RECORD_AUDIO" />
-    <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```
Comment (mathewnik90) [inline app/src/main/AndroidManifest.xml:7]: Did we face any issues after removing the permissions?

File: app/src/main/config/settings_diagnostics.plist [hunk 3]
```diff
@@ -118,7 +118,7 @@
 	<string>false</string>
 	<!--  -->
 	<key>startup_request_device_owner</key>
-	<string>true</string>
+	<string>false</string>
```
Comment (mathewnik90) [inline app/src/main/config/settings_diagnostics.plist:121]: Why are all these parameters changed to false?

**Review by mathewnik90 — CHANGES_REQUESTED** [review]
Please take a look at the comments given below

File: app/src/main/config/settings_pedro.plist (hunk 1 above)
Comment (akashdabhi03) [reply app/src/main/config/settings_pedro.plist:121]: If set to true, the app prompts the user to allow "Modify system settings" permission.

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (akashdabhi03) [reply app/src/main/AndroidManifest.xml:7]: No we did not face any issues, as of yet, during testing.

File: app/src/main/config/settings_diagnostics.plist (hunk 3 above)
Comment (akashdabhi03) [reply app/src/main/config/settings_diagnostics.plist:121]: Same reason as above.

**Comment by akashdabhi03** [comment]
> Please take a look at the comments given below

I have addressed all the comments. Please review and let me know in case you require additional information.

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**